src/MoBi.UI/Services/ToolTipCreator.cs

```diff
@@ -13,7 +14,7 @@ namespace MoBi.UI.Services
    public interface IToolTipCreator : OSPSuite.UI.Services.IToolTipCreator
    {
       SuperToolTip ToolTipFor(NotificationMessage notificationMessage);
-      SuperToolTip ToolTipFor(IMoBiParameterDTO parameterDTO);
+      SuperToolTip ToolTipFor(IMoBiParameterDTO parameterDTO, string cellValue = null);
```

Comment:
this `cellValue` parameter name is very confusing to me. What is a cell in the context of a parameter? I think this should be renamed and ideally, an example added if the context is hard to explaijn

Reply:
The tooltip is at a gridcell, and this cell has a value / text. The underlaying object is a parameter, but it has a lot of properties. You could argue that it is thus the property value that matters, but only the grid cell displays the value including the correct formatting. Hence, I chose the name cellValue